rendered.yml

```diff
@@ -18,4 +18,4 @@ default_context:
   use_read_the_docs: "y"
   sphinx_theme: "astropy-boostrap"
   initialize_git_repo: "n"
-  minimum_python_version: 2.7
+  minimum_python_version: "2.7"
```

Comment:
have it 3.5 ;)

Reply:
Not 3.6? :wink:

Reply:
We can change this when we change the .travis yml in the template, for now let's get it working please!